packs/data/equipment-effects.db/effect-cognitive-mutagen-greater.json

```diff
@@ -19,41 +19,23 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "selector": "arcana",
-                "type": "item",
-                "value": 3
-            },
-            {
-                "key": "FlatModifier",
-                "selector": "crafting",
-                "type": "item",
-                "value": 3
-            },
-            {
-                "key": "FlatModifier",
-                "selector": "occultism",
-                "type": "item",
-                "value": 3
-            },
-            {
-                "key": "FlatModifier",
-                "selector": "society",
+                "selector": [
+                    "arcana",
+                    "crafting",
+                    "lore-skill-check",
+                    "occultism",
+                    "society"
+                ],
```

Comment:
Looks like these were done wrong anyways. The lore skill check should be predicated on `action:recall-knowledge`. Because Paizo loves to make our lives harder there is at least one official lore skill that is not used to recall knowledge (Piloting Lore, https://2e.aonprd.com/Feats.aspx?ID=3140). All of the cognitive mutagens will need to get fixed.

Reply:
Yeah, that's probably best. Once this is merged I'll add a ticket, then start on it.